scope & closures/ch5.md

```diff
@@ -1,15 +1,15 @@
-# You Don't Know JS: Scope & Closures
-# Chapter 5: Scope Closure
+# You Don't Know JS: Escopos e Clausuras
+# Capítulo 5: Scope Closure
```

Comment:
"Escopo & _Closures_" ou "Escopo e _Closures_". 

Manutenção da palavra em sua língua original. Discutido na [issue 91](https://github.com/cezaraugusto/You-Dont-Know-JS/issues/91).